config/zephyr/Kconfig

```diff
@@ -34,6 +34,45 @@ menuconfig CHIP
 
 if CHIP
 
+config CHIP_DEVICE_VENDOR_ID
+	int "Device vendor ID"
+	default 9050 # 0x235A
+	range 0 65535
+	help
+	  Device vendor ID is used by various CHIP components. For example, it is
+	  an attribute of the Basic Information cluster and it is included in OTA
+	  (Over-the-air update) image header.
+
+config CHIP_DEVICE_PRODUCT_ID
+	int "Device product ID"
+	default 0
+	range 0 65535
+	help
+	  Device product ID is used by various CHIP components. For example, it is
+	  an attribute of the Basic Information cluster and it is included in OTA
+	  (Over-the-air update) image header.
+
+config CHIP_DEVICE_SOFTWARE_VERSION
+	int "Device software version"
+	default 0
+	range 0 4294967295
+	help
+	  Device software version is used by various CHIP components. For example,
+	  it is an attribute of the Basic Information cluster and it is included in
+	  OTA (Over-the-air update) image header. Note that due to the rollback
+	  protection a device will only accept software updates with versions
+	  greater than its current one.
+
+config CHIP_DEVICE_SOFTWARE_VERSION_STRING
```

Comment:
According to spec the software version string attribute in BasicInformationCluster maximum length is 64 bytes of UTF-8 characters. I'm not sure if Kconfig can do such check, but maybe we could at least add this information to help?

Reply:
Ok. I'll add.

Reply:
Done